### ISSUE TITLE
sbfl_method_ranks_full.json

### ISSUE THREAD

**Issue by Seanmc1005** [coinse/autofl_defects4j_data#1]
I'm sorry to bother you. I saw that you used the file "sbfl_method_ranks_full.json". Could you please give a detailed description of how to construct this file? Looking forward to your reply. Thanks.

**Comment by Seanmc1005** [comment]
Also, the data set you are using here is defects4j-1.0.0, but this data set does not have the command "defects4j export -p classes.relevant -o classes.relevant". Are you doing this on a higher version of Defects4J?

**Comment by agb94** [comment]
Yes, we used the latest version of the Defects4J framework, but we specifically utilized the bugs from version 1.0.0 for easier comparisons with other techniques. The script in this repository measures Defects4J coverage using GZoltar, which can then be used to calculate the SBFL scores in the json file.

**Comment by Seanmc1005** [comment]
> 是的，我们使用了最新版本的 Defects4J 框架，但我们特别利用了 1.0.0 版本中的错误，以便于与其他技术进行比较。此存储库中的脚本使用 GZoltar 测量 Defects4J 覆盖率，然后可以使用该脚本计算 json 文件中的 SBFL 分数。

Thank you very much for your reply, but when I directly used your script run_gzoltar, I found that a lot of data in the Mockito project could run normally, but the final matrix and other files were empty. Have you encountered this problem? Thank you for your reply.

**Comment by agb94** [comment]
Umm I haven't. Is this only happening to Mockito?

**Comment by Seanmc1005** [comment]
Sorry, I only tried the Mockito project outside of your original data set. At the same time, I directly ran your gzoltar.sh script, that is, only obtained the coverage data of failed unit tests. I found that the matrix of Chart-8 showed a '+' symbol in the last column, which was inconsistent with the fact that it was a failed test case. Have you ever been in this situation? Thank you very much for your reply.
[Chart_8.tar.gz](https://github.com/user-attachments/files/18208404/Chart_8.tar.gz)
